### PR TITLE
A little copy editing

### PR DESCRIPTION
**Pull request by ilovezfs**

### CODE REVIEW

File: cmd/brew-services.rb [hunk 1]
```diff
@@ -46,21 +46,21 @@
...
 #    { :url => "https://gist.github.com/raw/534777/63c4698872aaef11fe6e6c0c5514f35fd1b1687b/nginx.plist.xml" }
 #
-# Some simple template parsing is performed, all variables like `{{name}}` are
+# Some simple template parsing is performed. All variables like `{{name}}` are
 # replaced by basically doing:
 # `formula.send('name').to_s if formula.respond_to?('name')`, a bit like
-# mustache. So any variable in the `Formula` is available as template
-# variable, like `{{var}}`, `{{bin}}` usw.
+# mustache. So any variable in the `Formula` is available as a template
+# variable, like `{{var}}`, `{{bin}}`, and `{{usr}}`.
```
Comment (ilovezfs) [inline cmd/brew-services.rb]: This was just a guess. I have no idea what was actually intended by "usw."

Comment (mikemcquaid) [inline cmd/brew-services.rb]: :+1:

File: cmd/brew-services.rb [hunk 2]
```diff
@@ -13,23 +13,23 @@
 #
 # ## DESCRIPTION
 #
-# Integrates homebrew formulae with MacOS X' `launchctl` manager. Services
-# can either be added to `/Library/LaunchDaemons` or `~/Library/LaunchAgents`.
-# Basically items added to `/Library/LaunchDaemons` are started at boot,
-# those in `~/Library/LaunchAgents` at login.
+# Integrates Homebrew formulae with Mac OS X' `launchctl` manager. Services
```
Comment (mikemcquaid) [inline cmd/brew-services.rb]: Can just be `OS X's`

**Comment by mikemcquaid** [comment]
One comment otherwise: :bow for fixing this.

**Comment by mikemcquaid** [comment]
@ilovezfs Happy to merge when the comments are addressed?

**Comment by ilovezfs** [comment]
@mikemcquaid Refreshed, including a few more tweaks and your change.

**Comment by mikemcquaid** [comment]
Thanks @ilovezfs!

**Comment by ilovezfs** [comment]
@mikemcquaid You're welcome! Next up, style fixes.

**Comment by mikemcquaid** [comment]
@ilovezfs I recommend `brew style --fix` to address some of them automatically.

**Comment by ilovezfs** [comment]
Wait, but isn't that cheating? :) Thanks!

**Pull request merged**